packages/flutter_tools/lib/src/commands/update_packages.dart

```diff
@@ -144,6 +145,7 @@ class UpdatePackagesCommand extends FlutterCommand {
           exitCode: 1,
         );
       }
+      printStatus('All pubspecs were up to date.');
       return;
     }
 
```

Comment:
cc @jonahwilliams for review of the changes in this file

Reply:
LGTM